udata_front/views/dataset.py

```diff
@@ -139,3 +142,16 @@ def group_resources_by_type(resources):
         if groups[rtype]:
             ordered[(rtype, str(rtype_label))] = groups[rtype]
     return ordered
+
+
+@Dataset.on_update.connect
+def clear_cache_on_updated_dataset(dataset):
+    user = current_user.slug or 'anonymous'
+
+    for lang_code in current_app.config['LANGUAGES'].keys():
+        head_cache_key = make_template_fragment_key("dataset-extra-head", vary_on=[str(dataset.id), lang_code])
+        cache.delete(head_cache_key)
+        breadcrumb_cache_key = make_template_fragment_key("dataset-breadcrumb", vary_on=[str(dataset.id), lang_code])
+        cache.delete(breadcrumb_cache_key)
+        content_cache_key = make_template_fragment_key("dataset-content", vary_on=[str(dataset.id), "1", lang_code, user])
```

Comment:
The template cache here is defined in the jinja file as following:
`{% cache cache_duration, 'dataset-content', dataset.id|string, reuses.page|string, g.lang_code, current_user.slug or 'anonymous' %}`

The `reuses.page` is not know in the event context. Should we remove this key? Knowing the keys `dataset-content`, `dataset.id`, `g.lang_code` and `current_user.slug` might be enough?